Services/UserService.cs

```diff
@@ -34,26 +34,26 @@ public UserService(IOptions<DirectoryConfig> config)
             var options = new AsertoDirectoryOptions(opts.ServiceUrl, opts.APIKey, opts.TenantID, opts.Insecure);
 
             var optionsInt = Microsoft.Extensions.Options.Options.Create(options);
-            directoryClient = new DirectoryAPIClient(optionsInt, loggerFactory);
+            directoryClient = new Aserto.AspNetCore.Middleware.Clients.Directory.V3.Directory(optionsInt, loggerFactory);
         }
 
         private async Task<GetUserResponse> GetUserBySub(string sub)
         {
             try
             {
-                var getRelationResponse = await directoryClient.GetRelationAsync(subjectType: "user", objType: "identity", objKey: sub, relationName: "identifier", relationObjectType: "identity");
-                if (getRelationResponse.Results.Count == 0)
+                var getRelationResponse = await directoryClient.GetRelationAsync(subjectType: "user", objType: "identity", objId: sub, relationName: "identifier");
+                if (getRelationResponse.Result == null)
```

Comment:
I don't think `Result` can be null. Wouldn't you get back an error if the relation isn't found?